sites/docs/src/content/docs/usage/configuration/system_requirements.md

```diff
@@ -0,0 +1,121 @@
+---
+title: System requirements
+subtitle: Configure your system requirements
+shortTitle: System requirements
+weight: 3
+---
+
+## System requirements
+
+nf-core pipelines are reproducible, portable, and scalable, and are designed to work "out of the box". However, it is likely you will need to configure each pipeline to suit your data and system requirements.
+
+The following sections are common considerations for nf-core pipeline users.
+
+### Executors
+
+Nextflow pipelines will run locally by default.
+Most users will need to specify an "executor" to submit jobs to a scheduler (e.g. SGE, LSF, SLURM, PBS, and AWS Batch).
+
+Schedulers can be configured using scopes settings that can be specified using [shared configuration profiles](#shared-configuration-files) or [custom configuration files](#custom-parameter-and-configuration-files).
```

Comment:
```suggestion
Most users will need to specify an "executor" to submit jobs to a scheduler (e.g. SGE, LSF, SLURM, PBS, and AWS Batch).

A simple example of a config file specifying this could be:

    ```nextflow
    process {
        executor = 'slurm'
        queue = { task.time >= '24.d' ? 'long' : 'short' }
    }
    ```

For more complex examples, see the various institutional configs on the [nf-core/configs repository](https://github.com/nf-core/configs/tree/master/conf)
```